dataSourceImpl/src/main/java/QuizGatewayImpl.kt

```diff
@@ -0,0 +1,14 @@
+class QuizGatewayImpl() : QuizGateway {
+    override fun getSessionToken(): String {
+        TODO("Not yet implemented")
+    }
+
+    override fun getQuestions(
+        amount: Int,
+        category: String,
+        difficulty: String,
+        type: String
+    ): String {
+        TODO("Not yet implemented")
+    }
+}
```

Comment:
Avoid creating clases with empty implementations. In this case just remove this file if it has empty implementations

Reply:
+